clients/feeder/testdata/sepolia/block/485004.json

```diff
@@ -0,0 +1,2040 @@
+{
+    "block_hash": "0x7f9413a02d787cb52046925414b4f4f4cfde93b4a96f92fa7c57eb0ac302f3",
```

Comment:
Where did you get this test data? As far as I can see Sepolia block height is ~56,000 not 485,004

Reply:
Oh right, you just copied the Goerli block data. I think you should add new test data, eg get a Sepolia block + signature

Reply:
https://alpha-sepolia.starknet.io/feeder_gateway/get_block?blockNumber=56377

Reply:
https://alpha-sepolia.starknet.io/feeder_gateway/get_signature?blockNumber=56377

Reply:
You should do the same for your other issue https://github.com/NethermindEth/juno/pull/1793